Packages/Newtonsoft.Json-for-Unity.Converters/Configuration/UnityConvertersConfig.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Collections.Generic;
+using Newtonsoft.Json.Converters;
+using UnityEngine;
+
+namespace Newtonsoft.Json.UnityConverters.Configuration
+{
+#pragma warning disable CA2235 // Mark all non-serializable fields
+    [Serializable]
+    public sealed class UnityConvertersConfig : ScriptableObject
+    {
+        internal const string PATH = "Assets/Resources/Newtonsoft.Json-for-Unity.Converters.asset";
+        internal const string PATH_FOR_RESOURCES_LOAD = "Newtonsoft.Json-for-Unity.Converters";
+
+        public bool useUnityContractResolver = true;
+
+        public bool useAllOutsideConverters = true;
+
+        public List<ConverterConfig> outsideConverters = new List<ConverterConfig>();
```

Comment:
Codacy found an issue: [Make 'outsideConverters' private.](https://app.codacy.com/manual/jilleJr/Newtonsoft.Json-for-Unity.Converters/pullRequest?prid=5865055)